tox.ini

```diff
@@ -40,20 +38,15 @@ passenv =
     PYTEST_ADDOPTS
 commands = pytest {posargs:tests/functional/}
 
-[testenv:functional-py3]
-basepython = python3.6
-skip_install = true
-deps =
-    pytest
-    webtest
-    factory-boy
-    -rrequirements.txt
-passenv =
-    BROKER_URL
-    ELASTICSEARCH_URL
-    TEST_DATABASE_URL
-    PYTEST_ADDOPTS
-commands = pytest {posargs:tests/functional/}
+[testenv:functional]
+deps = {[functional]deps}
+passenv = {[functional]passenv}
+commands = {[functional]commands}
+
+[testenv:functional-py36]
+deps = {[functional]deps}
+passenv = {[functional]passenv}
+commands = {[functional]commands}
```

Comment:
This is the same inheritance thing that our docs envs below are already doing. There's no way to say to tox "just inherit everything from `functional`" as far as I know, you have to explicitly inherit each setting (deps, passenv, commands) like this.